packages/ui/components/accordion/src/LionAccordion.js

```diff
@@ -210,10 +210,10 @@ export class LionAccordion extends LitElement {
    */
   __rearrangeInvokersAndContent() {
     const invokers = /** @type {HTMLElement[]} */ (
-      Array.from(this.querySelectorAll('[slot="invoker"]'))
+      Array.from(this.querySelectorAll(':scope > [slot="invoker"]'))
```

Comment:
just as a consideration - in tabs we filter on children... potentially faster
https://github.com/ing-bank/lion/blob/master/packages/ui/components/tabs/src/LionTabs.js#L170